client/extensions/woocommerce/state/README.md

```diff
@@ -0,0 +1,16 @@
+State
+=====
+
+This directory contains all of the behavior describing the application state for the woocommerce extension. This is a sub-tree of the global state, stored in `state.extensions.woocommerce`.
+
+## Structure
+
+### [`site`](site/README.md)
+
+### [`ui`](ui/README.md)
+
+Holds data changed in the UI but not yet saved
```

Comment:
I view the `ui` context as one that contains any extension UI states (potentially even whether or not accordions are expanded, etc.) This (for now) can include "edit states" of API data, but even that might be moving to its own tree as we expand its usage.